### DISCUSSION TITLE
Setting up a new laptop for iCub

### DISCUSSION THREAD

**Discussion by AndreyGalitsin** [robotology/community#552] [Q&A] [answered]
Hello, I want to set up a new laptop to use it for communication with iCub. Tell me please do you have a step by step tutorial how can I do it, what software should I install?

All the best,
Andrey

**Comment by maggia80** [comment, marked as answer]
Hi @AndreyGalitsin, you can find some documentation here:  https://icub-tech-iit.github.io/documentation/icub_operating_systems/other-machines/generic-machine/

**Reply by AndreyGalitsin** [reply]
I will try this, thank you!
